core/ir.mli

```diff
@@ -30,6 +30,8 @@ type 'a var_map = 'a Utility.intmap
 
 type language = string
 
+val var_of_binder : binder -> var
```

Comment:
No need to export this in `ir.mli`, since it already exists in `Var.var_of_binder`